redfish-core/include/redfish.hpp

```diff
@@ -82,8 +82,11 @@ class RedfishService
         requestRoutesThermal(app);
         requestRoutesPower(app);
 #endif
-        requestRoutesThermalSubsystem(app);
-        requestRoutesThermalMetrics(app);
```

Comment:
It looks like this drops ThermalSubsystem? Why the changes here?

Reply:
We should not remove the ThermalSubsystem and ThermalMetrics.